crates/op-rbuilder/src/primitives/bundle.rs

```diff
@@ -9,19 +9,30 @@ pub struct Bundle {
     #[serde(rename = "txs")]
     pub transactions: Vec<Bytes>,
 
+    #[serde(rename = "reverting_tx_hashes")]
+    pub reverted_hashes: Vec<B256>,
```

Comment:
```suggestion
    pub reverting_hashes: Vec<B256>,
```

to make it consistent with the serde name